### PR TITLE
Rikudo830 patch 1

### PR DESCRIPTION
**Pull request by rikudo830**
## Sony apps  from request form and more icons added

Fixes #(issue)

## Type of change

:x: Bug fix (non-breaking change which fixes an issue)
:x: Icon addition (non-breaking change that adds/modifies Lawnicons's icons)
:x: General change (non-breaking change that doesn't fit the above categories, such as copyediting)

### Icons added
* Music(`com.sonyericsson.music) - added from icon request form
* Photo Pro (com.sonymobile.photopro`)
* Video Pro (`jp.co.sony.mc.videopro)
* South Park (com.ubisoft.dragonfire)
* SmartBand Talk (com.sonymobile.hostapp.swr30)
* Root Explorer (com.speedsoftware.rootexplorer)
* Music Pro (jp.co.sony.mc.musicpro)
* Cinema Pro (com.sonymobile.cinemapro)
* Game enhancer (com.sonymobile.gameenhancer)

### CODE REVIEW

**Comment by KTibow** [comment]
also: instead of `com.sonyericsson.music.svg`, you need to call it `sony_music.svg`.

File: svgs/com.sonymobile.hostapp.swr30.svg
Comment (KTibow) [inline svgs/com.sonymobile.hostapp.swr30.svg:1]: the way the clock circle inside the watch is placed feels a bit off.
this also doesn't match the original icon. maybe i'm missing something but could you explain why you drew the watch this way?

File: svgs/com.sonymobile.photopro.svg
Comment (KTibow) [inline svgs/com.sonymobile.photopro.svg:1]: we can reuse the existing camera drawable.

File: svgs/com.sonyericsson.music.svg
Comment (KTibow) [inline svgs/com.sonyericsson.music.svg:1]: this doesn't match the original icon. the original icon has more circular bottom things and isn't slanted. is this icon reused or something?

File: svgs/com.sonymobile.cinemapro.svg
Comment (KTibow) [inline svgs/com.sonymobile.cinemapro.svg:1]: this doesn't match the original icon (from what I can find on google). is this icon reused or something?

File: svgs/com.speedsoftware.rootexplorer.svg
Comment (KTibow) [inline svgs/com.speedsoftware.rootexplorer.svg:1]: this is not the xiaomi file explorer, so don't reuse the xiaomi file explorer logo.

File: svgs/com.ubisoft.dragonfire.svg
Comment (KTibow) [inline svgs/com.ubisoft.dragonfire.svg:1]: this doesn't match the original icon at all.

**Comment by KTibow** [comment]
@rikudo830 just to remind you - as of right now, the way you added these icons actually breaks the process that builds Lawnicons, and there are a couple other things you need to correct before this PR is merged.

File: svgs/com.sonymobile.photopro.svg
Comment (Chefski) [inline svgs/com.sonymobile.photopro.svg:1]: Duplicate of `camera.svg`. Remove the icon and connect the component to `camera.svg`

File: svgs/com.speedsoftware.rootexplorer.svg
Comment (Chefski) [inline svgs/com.speedsoftware.rootexplorer.svg:1]: Duplicate of `mi_files.svg`. Remove the icon and connect the component to `mi_files.svg`.

File: svgs/jp.co.sony.mc.videopro.svg
Comment (Chefski) [inline svgs/jp.co.sony.mc.videopro.svg:1]: Looks like a duplicate of `screen_recorder.svg`.

**Review by Chefski — COMMENTED** [review]
Are some of these icons simply copied and renamed? Many of these exist looking almost/exactly the same which means they don't need to be added again. Also please use `_` instead of `.` in the svg file names and shorten them to simply the name of the app.

**Comment by x9136** [comment]
In this case, it's better to look at 2 PRs at once. #1412

**Comment by Chefski** [comment]
@x9136 is it duplicate PRs??

**Comment by x9136** [comment]
> @x9136 is it duplicate PRs??

Partial.

**Comment by KTibow** [comment]
imo they should just use one branch (and to extend on that, one PR), and if they don't know how to do that someone else should make their PR

**Pull request closed without merging**